CMakeLists.txt

```diff
@@ -60,6 +60,13 @@ if (UTF8PROC_INSTALL)
     FILES
       "${PROJECT_SOURCE_DIR}/utf8proc.h"
     DESTINATION include)
+
+  set(PREFIX ${CMAKE_INSTALL_PREFIX})
+  set(LIBDIR lib)
+  set(INCLUDEDIR include)
+  set(VERSION 2.6.1)
+  configure_file(libutf8proc.pc.in  libutf8proc.pc @ONLY)
+  install(FILES ${CMAKE_BINARY_DIR}/libutf8proc.pc DESTINATION lib/pkgconfig)
```

Comment:
Hardcoded destination feels wrong. You should use GNUInstallDirs so downstream users don't have to patch `lib` to be `lib64`

Reply:
Kind of dependent on https://github.com/JuliaStrings/utf8proc/pull/159/files I guess.

Reply:
Thank you for your suggestion. I have changed it.

Reply:
I've merged #159, so one can use that now.